tests/test_pfs.py

```diff
@@ -39,15 +39,15 @@ def test_delete_non_existent_repo():
     assert len(client.list_repo()) == orig_repo_count
 
 
-#def test_delete_all_repos():
-#    client = python_pachyderm.Client()
+def test_delete_all_repos():
```

Comment:
Hmm I'm quite sure this test was failing when we were trying to merge the v7.x PR.

Reply:
Yeah, this test currently fails due to an apparent bug in pachyderm/pachyderm. I filed an issue there and linked to it in the comments